### ISSUE TITLE
A way to get the project ID

### ISSUE THREAD

**Issue by gzuidhof** [hetznercloud/hcloud-go#451]
### TL;DR

Given an API key, expose some endpoint / routine to get the project ID number.

### Expected behavior

Hi `hcloud-go` team,

I built [this tool](https://github.com/gzuidhof/flipper) that automatically re-points floating IPs to healthy servers. One of the core features is that it notifies a Slack or Mattermost channel.

Right now it will print a list of servers, floating IPs, and their health state - which is great. It would be really great if I were able to provide a link to the resources as well - but to construct the URL I would need the project ID.

I'd prefer not to have a config field for this and rather have it figure that out on its own, is there any way to do that?

Thank you!

**Comment by jooola** [comment]
Hi @gzuidhof 

We do not provide a public API to retrieve the current project ID.

I'll add your request to our internal feedback tracker regarding making the project API open to the public.

**Comment by gzuidhof** [comment]
Thank you for the response :) I'll make it a config field until perhaps that changes.
